.github/workflows/rust.yml

```
@@ -50,55 +48,47 @@ jobs:
           override: true
       - uses: actions-rs/cargo@v1
         with:
-          command: check 
+          command: check
           args: --all-features
 
   test_all_features:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+
+      - name: Start RabbitMQ server
+        run: docker-compose up -d
```

you can call the updated script like below
```suggestion
        run: /start_rabbitmq.sh
```

---

Oh I see. It was only using the container before.